app/src/org/commcare/utils/StringUtils.java

```diff
@@ -75,4 +75,29 @@ public static String getLocalizedLevel(String levelCode, Context context) {
             default -> levelCode;
         };
     }
+
+
+    public static int convertToIntegers(String input) {
+        int returnInt = 0;
+        if (input == null || input.isEmpty()) {
+            return returnInt;
+        }
+
+        try {
+            for (int i = 0; i < input.length(); i++) {
+                char ch = input.charAt(i);
+                if (Character.isAlphabetic(ch)) {
+                    char lowerCaseCh = Character.toLowerCase(ch);
+                    returnInt += lowerCaseCh - 'a' + 1; // 'a' is 1, 'b' is 2, etc.
+                } else if (Character.isDigit(ch)) {
+                    returnInt += ch - '0';
+                } else {
+                    returnInt += 1;    // any random number
+                }
+            }
+        } catch (Exception ignore) {
+
+        }
+        return returnInt;
+    }
```

Comment:
Have we explored other options for generating IDs (such as `UUID.randomUUID()` from the Java Util library and concatenating that with a string like `"notification"` for example)?